src/components/EventItem.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Dotdotdot from 'react-dotdotdot';
+import { capitalizeFirst } from 'utils';
+
+export const EventItem = ({
+  id,
+  name,
+  description,
+  eventDate,
+  timeRange,
+  place,
+}) => (
+  <div key={id} className="max-w-md mb-2">
+    <div className="px-6 py-4">
+      <div className="font-bold text-xl mb-1">
+        <a href={`https://www.facebook.com/events/${id}/`}>{name}</a>
```

Comment:
`id` is not a correct parameter:

https://www.facebook.com/events/1365d631-8c0a-5b49-9ebd-9eb38b361bc6/

![Screenshot 2019-04-06 at 14 30 59](https://user-images.githubusercontent.com/1255989/55669531-ac353a00-5878-11e9-81db-7d7764363b30.png)

Reply:
Hmm... graphql's node ID is overwriting FB event ID.